frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -81,6 +82,73 @@ const DataConsumptionCard: React.FC<DashboardItemProps> = ({
     if (suffixLabel) suffixLabel = `(in ${suffixLabel})`;
   }
 
+  if (isLoading) {
+    body = (
+      <>
+        <div className="skeleton-text nb-data-consumption-card__chart-skeleton" />
+        <GraphEmpty height={200} loading />
+        <div className="skeleton-text nb-data-consumption-card__chart-legend-skeleton" />
+      </>
+    );
+  } else if (!error && !chartData.some(_.isEmpty)) {
+    body = (
+      <>
+        <div className="nb-data-consumption-card__chart-label text-secondary">
+          {CHART_LABELS[sortByKpi]} {suffixLabel}
+        </div>
+        <Chart
+          containerComponent={
+            <ChartVoronoiContainer
+              labelComponent={<ChartTooltip style={{ fontSize: 8, paddingBottom: 0 }} />}
+              labels={({ datum }) => `${datum.y} ${maxUnit}`}
+              voronoiDimension="x"
+            />
+          }
+          minDomain={{ y: 0 }}
+          maxDomain={{ y: maxVal + 10 }}
+          domainPadding={{ x: [padding, padding] }}
+          legendComponent={
+            <ChartLegend
+              themeColor={ChartThemeColor.purple}
+              data={legendData}
+              orientation="horizontal"
+              symbolSpacer={5}
+              gutter={2}
+              height={30}
+              padding={{ top: 50, bottom: 0 }}
+              style={{ labels: { fontSize: 8 } }}
+            />
+          }
+          legendPosition="bottom-left"
+          padding={{
+            bottom: 50,
+            left: 30,
+            right: 20,
+            top: 30,
+          }}
+          themeColor={ChartThemeColor.purple}
+        >
+          <ChartAxis style={{ tickLabels: { fontSize: 8, padding: 2 } }} />
+          <ChartAxis
+            dependentAxis
+            showGrid
+            tickCount={10}
+            style={{
+              tickLabels: { fontSize: 8, padding: 0 },
+            }}
+          />
+          <ChartGroup offset={sortByKpi === BY_EGRESS ? 0 : 11}>
+            {chartData.map((data, i) => (
+              <ChartBar key={`chartbar-${i}`} data={data} /> // eslint-disable-line react/no-array-index-key
```

Comment:
```suggestion
              <ChartBar key={`chartbar-${i}`} data={data} />
```
No need to disable the rule

Reply:
Required, since its still using array index.
May be we can refactor it in 4.6 to send in the indexes in chartdata.
Lets keep the PR focused to what bug its trying to solve.

Reply:
Sure. But I have passed keys like the way you have passed it in several places and it dint give me that linter error. But if you're facing the issue then we need to disable it. No issues. :)